src/ripple/app/tx/impl/ApplyContext.h

```diff
@@ -23,6 +23,7 @@
 #include <ripple/ledger/ApplyViewImpl.h>
 #include <ripple/core/Config.h>
 #include <ripple/protocol/STTx.h>
+#include <ripple/protocol/XRPAmount.h>
```

Comment:
I think we might want to just add this to `<ripple/protocol/st.h>` and use that include instead of individually picking out items.